### DISCUSSION TITLE
Performance issues in Filament when displaying large datasets in a Resource List View

### DISCUSSION THREAD

**Discussion by carstenlehmann** [filamentphp/filament#8431] [Help]
Hello dear community,

I am currently using Laravel Filament for the development of a project and so far, I am very satisfied with it. However, I have encountered a performance issue when trying to display a large number of datasets. The loading time is extremely slow.

I have already checked the database queries and found that they have no impact on the performance. The problem seems to lie elsewhere. Particularly interesting is the observation that the count of rendered views in the Laravel Debugbar is extremely high.

Has anyone encountered this issue and possibly found a solution for it? Are there any ways to improve performance in Filament when displaying large datasets? I would greatly appreciate your help and experiences with this.

The only way to improve the loading-time is to paginate only 10 items... But there is clearly a better way or am i wrong?

Thank you in advance!

Best regards,

Carsten

**Comment by leandrocfe** [comment]
I believe it would be helpful if you could provide some examples or share a GitHub repository that showcases the issue you're facing. This way, someone can offer valuable tips to enhance the performance.

**Reply by carstenlehmann** [reply]
i will try to do a clean test, new laravel, new filament, just one model with test-data. If i can reproduce the issue there, i can share the code.

**Comment by robertorinaldi-dev** [comment]
Try without debugbar
